### PR TITLE
Adds "air" at the bottom of the page

### LINKED ISSUE #4263: Configuration menu not properly finished in advanced mode
**Checklist:**

- [x] I updated to the latest version available
- [x] I cleared the cache of my browser

**Home Assistant release with the issue:**
0.102.0

**Last working Home Assistant release (if known):**
Not sure, but I suspect 0.101.3 since I never noticed before. 

**UI (States or Lovelace UI?):**
Lovelace UI

**Browser and Operating System:**
Google Chrome on Android 9
Mozilla Firefox on Ubuntu 19.10

**Description of problem:**
When opening the configuration menu and having advanced mode enabled, the bottom configuration menu isn't finished nicely. In the image below, I'm scrolled all the way down. 

Screenshot_20191121-190606~3

**Additional information:**
This does not happen in Desktop mode using Google Chrome mobile. It also doesn't happen when advanced mode is disabled.

### PR DESCRIPTION
**Pull request by ludeeus**
Fixes #4263 
image

### CODE REVIEW

**Comment by bramkragten** [comment]
Won't this add a margin to both the elements? When only want to target the last one?

File: src/panels/config/dashboard/ha-config-dashboard.ts [hunk 1]
```diff
@@ -138,6 +138,9 @@ class HaConfigDashboard extends LitElement {
     return [
       haStyle,
       css`
+        ha-config-navigation {
```
Comment (bramkragten) [inline src/panels/config/dashboard/ha-config-dashboard.ts]:
```suggestion
        ha-config-navigation:last-child {
```

**Comment by bramkragten** [comment]
And check if the promo banner also needs some "air"

**Comment by ludeeus** [comment]
> Won't this add a margin to both the elements?

Make sense, but it did not look off, `:last-child` are probably better

**Comment by ludeeus** [comment]
Added to promo too:
image
vs
image

**Review by bramkragten — APPROVED** [review]

**Pull request merged**